packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/page-banner-as-white.md

```diff
@@ -0,0 +1,6 @@
+Description: This is a variant of the [Page Banner](./?p=organisms-page-banner) showing an example with a white background.
```

Comment:
Please consolidate this doc into page-banner.md
All variants should be documented into the main pattern doc

Reply:
done.